### DISCUSSION TITLE
Hero 11 Mini new lab commands on Hero 11

### DISCUSSION THREAD

**Discussion by kanishk619** [gopro/labs#388] [Q&A] [answered]
Just wanted to know when the new lab commands will be available for the Hero 11? It looks like certain good commands have been added to 11 mini based on changelog. Really looking forward for MFAST and !X on Hero 11

**Comment by dnewman-gpsw** [comment, marked as answer]
There were just more main line firmware updates scheduled for Mini, so it got these Labs enhancement first, but they will be made available on HERO11 and 10 (hopefully), later this year.
